_includes/includes/head.php

```diff
@@ -30,8 +30,13 @@
   }
 ?>
 <!DOCTYPE html>
-
-<html>
+<?php
+  if(!isset($language) || empty($language)) {
```

Comment:
```suggestion
  if(empty($language)) {
```

empty also tests isset ... https://www.php.net/manual/en/function.empty.php